README.md

```diff
@@ -146,6 +146,10 @@ For a debug build:
 Test
 ----
 
+If a ruby was found during configuration, execute the following command before running tests:
```

Comment:
Yeah, I can live with a little Windows specific thing for now rather than creating a new target or doing some other tomfoolery.